package.json

```diff
@@ -34,124 +34,123 @@
-    "d3-drag": "^2.0.0",
-    "d3-dsv": "^2.0.0",
-    "d3-format": "^2.0.0",
-    "d3-geo": "^2.0.1",
+    "d3-array": "^3.1.1",
+    "d3-axis": "^3.0.0",
+    "d3-brush": "^3.0.0",
+    "d3-color": "^3.0.1",
+    "d3-drag": "^3.0.0",
+    "d3-dsv": "^3.0.1",
+    "d3-format": "^3.1.0",
+    "d3-geo": "^3.0.1",
     "d3-queue": "^3.0.7",
     "d3-request": "^1.0.6",
     "d3-scale": "^4.0.2",
-    "d3-selection": "^2.0.0",
+    "d3-selection": "^3.0.0",
     "d3-transition": "^3.0.1",
     "d3-zoom": "^3.0.0",
-    "dom-scroll-into-view": "^1.2.1",
-    "enzyme-adapter-react-16": "^1.15.6",
-    "eventemitter3": "^2.0.2",
+    "dom-scroll-into-view": "^2.0.1",
+    "eventemitter3": "^4.0.7",
     "genbank-parser": "^1.0.0",
     "ndarray": "^1.0.18",
-    "pako": "^1.0.10",
+    "pako": "^2.0.4",
     "path": "^0.12.7",
-    "pixi.js": "^5.0.3",
+    "pixi.js": "^6.2.1",
     "prismjs": "^1.16.0",
     "prop-types": "^15.6.0",
-    "pub-sub-es": "^1.2.1",
-    "react": "^16.8.0",
-    "react-bootstrap": "0.32.1",
+    "pub-sub-es": "^2.0.1",
+    "react": "^17.0.2",
+    "react-bootstrap": "^2.1.1",
     "react-checkbox-tree": "^1.4.1",
     "react-color": "^2.13.8",
-    "react-dom": "^16.8.0",
+    "react-dom": "^17.0.2",
     "react-grid-layout": "^1.3.0",
-    "react-resizable": "^1.8.0",
+    "react-resizable": "^3.0.4",
     "react-simple-code-editor": "^0.11.0",
-    "react-sortable-hoc": "^1.10.1",
+    "react-sortable-hoc": "^2.0.0",
     "reactcss": "^1.2.3",
     "robust-point-in-polygon": "^1.0.3",
     "slugid": "^3.0.0",
-    "threads": "^0.7.2",
-    "transform-loader": "^0.2.3",
+    "threads": "^1.7.0",
+    "transform-loader": "^0.2.4",
     "unminified-webpack-plugin": "^3.0.0",
-    "url-join": "^1.1.0",
+    "url-join": "^4.0.1",
     "url-parse": "^1.4.3",
     "vkbeautify": "^0.99.3",
-    "whatwg-fetch": "^3.0.0"
+    "whatwg-fetch": "^3.6.2"
   },
   "devDependenciesComments": [
     "webpack-dev-server is pinned because later versions did not serve all static directories: https://github.com/higlass/higlass/issues/623"
   ],
   "devDependencies": {
-    "@babel/runtime": "^7.15.4",
     "@babel/core": "^7.15.8",
     "@babel/plugin-proposal-object-rest-spread": "^7.15.6",
     "@babel/plugin-transform-runtime": "^7.15.8",
     "@babel/polyfill": "^7.12.1",
     "@babel/preset-env": "^7.15.8",
     "@babel/preset-react": "^7.14.5",
-    "ajv-cli": "^3.0.0",
-    "autoprefixer": "^7.2.6",
+    "@babel/runtime": "^7.15.4",
+    "@wojtekmaj/enzyme-adapter-react-17": "^0.6.6",
+    "ajv-cli": "^5.0.0",
+    "autoprefixer": "^10.4.2",
     "babel-eslint": "^10.0.1",
     "babel-loader": "^8.2.2",
-    "babel-plugin-react-css-modules": "^3.4.2",
+    "babel-plugin-react-css-modules": "^5.2.6",
     "babel-preset-airbnb": "^5.0.0",
-    "brfs": "1.4.3",
+    "brfs": "^2.0.2",
     "browser-sync": "^2.24.7",
-    "chai": "^3.5.0",
-    "css-loader": "^0.28.8",
+    "chai": "^4.3.4",
+    "css-loader": "^3.6.0",
```

Comment:
It is not the latest major version, the latest is 6.5.1, but starting from 4.0.0 there are many breaking changes